src/plugins/mf_priority.cpp

```diff
@@ -1086,6 +1088,20 @@ static int inactive_cb (flux_plugin_t *p,
     // and look to see if any held jobs can be released
     b->cur_run_jobs--;
 
+    // also decrement the cur_node count for the association
+    if (R != NULL) {
+        std::string nodelist = extract_nodelist (R);
+        if (!nodelist.empty ()) {
+            int nnodes = process_nodelist (nodelist);
+            if (nnodes < 0)
+                flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB,
+                                             "mf_priority", 0,
+                                             "job.state.inactive: error "
+                                             "counting nnodes for job");
```

Comment:
Since the job is inactive, raising an exception probably won't do anything here.
(Also, the decrement of `b->cur_nodes` should be skipped if `nnodes < 0` to avoid instead _incrementing_ it. The same goes for where `b->cur_nodes` is incremented in the previous commit)

Actually I wonder if there's a good way for the plugin to save the `nnodes` for a job in the job aux item list so it doesn't even have to be recalculated here. Let me think on that.